src/main/resources/META-INF/rewrite/devcenter-starter.yml

```diff
@@ -57,3 +61,77 @@ recipeList:
   # in the Security DevCenter card.
   - io.moderne.devcenter.ReportAsSecurityIssues:
       fixRecipe: org.openrewrite.java.security.OwaspTopTen
+
+
+---
+# An example DevCenter configuration for a Quarkus application, with a named custom upgrade recipe for Quarkus
+type: specs.openrewrite.org/v1beta/recipe
+name: io.moderne.devcenter.DevCenterStarterQuarkus
+displayName: DevCenter
```

Comment:
Is this shown in anyway? Should we have a different `displayName` for each of the recipes or keep this as is?

Reply:
None of these changes belong in `devcenter-starter.yml`. We can provide examples of Quarkus if we want, but these aren't being added to the default devcenter we use on app tenant.